boards/arm/cy8cproto_062_4343w/cy8cproto_062_4343w.yaml

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022 Cypress Semiconductor Corporation.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: cy8cproto_062_4343w
+name: CY8CPROTO-062-4343W PSoC 6 Wi-Fi BT Prototyping Kit
+type: mcu
+arch: arm
+ram: 288
+flash: 2048
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
```

Comment:
remove xtools, we've deprecated the support

Reply:
done